lib/core.ps1

```diff
@@ -145,19 +145,16 @@ function Invoke-Git {
 
     $proxy = get_config PROXY
     $git = Get-HelperPath -Helper Git
-    $arguments = $ArgumentList -join ' '
-    $cmd = "`"$git`" $arguments"
 
     if ($WorkingDirectory) {
-        $cmd = "`"$git`" -C `"$WorkingDirectory`" $arguments"
+        $ArgumentList = @('-C', $WorkingDirectory) + $ArgumentList
     }
-    $sb = [scriptblock]::Create("& $cmd")
 
     if([String]::IsNullOrEmpty($proxy) -or $proxy -eq 'none')  {
-        return Invoke-Command $sb
+        return & $git $ArgumentList
```

Comment:
```suggestion
        return & $git $ArgumentList
```

This should be `$ArgumentList` or `@ArgumentList`? I mean splatting.

Reply:
Isn't splatting used for hashtables? `$ArgumentList` here is just a string array. Nevertheless, I tried a few complex commands and `$ArgumentList` seems to work fine.

Reply:
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_splatting?view=powershell-5.1

I think you should use splatting here for clear command.

Reply:
Done